src/Select/Select.jsx

```diff
@@ -44,15 +45,20 @@ export function Select({
     labelContainerClasses += ` ${cssClass.LABEL_HIDDEN}`;
   }
 
+  let reactSelectClasses = cssClass.REACT_SELECT;
+  if (readOnly) {
+    reactSelectClasses += ` ${cssClass.READ_ONLY}`;
+  }
+
   // The label container must be returned after the ReactSelect otherwise it does not get displayed
   // in the browser.
   return (
     <div className={cssClass.CONTAINER}>
       <div id={id}>
         <ReactSelect
-          className={cssClass.REACT_SELECT}
+          className={reactSelectClasses}
           clearable={clearable}
-          disabled={disabled}
+          disabled={disabled || readOnly}
```

Comment:
To implement the `readOnly` option, I use `ReactSelect`'s built-in `disabled` option and then apply some additional styles.